packages/devextreme/js/__internal/grids/grid_core/m_utils.ts

```diff
@@ -173,6 +173,15 @@ function normalizeGroupingLoadOptions(group) {
   });
 }
 
+export const popupUtils = {
+  createDoneButton: (options) => ({
+    toolbar: 'bottom', location: 'after', shortcut: 'done', options,
+  }),
+  createCancelButton: (options) => ({
+    toolbar: 'bottom', location: 'after', shortcut: 'cancel', options,
+  }),
+};
```

Comment:
Let's rename these methods
Because the ```create_done/end_button``` method doesn't create any button at all :)\
And let's add type to the ```options``` argument

I suggest the next namings:
```suggestion
export const popupUtils = {
  getDoneButtonConfig: (options: IToolbarButtonOptions) => ({
    toolbar: 'bottom', location: 'after', shortcut: 'done', options,
  }),
  getCancelButtonConfig: (options: IToolbarButtonOptions) => ({
    toolbar: 'bottom', location: 'after', shortcut: 'cancel', options,
  }),
};
```

Reply:
I agree with renaming from button to option. But I don't agree that there is no creation. The object is being created